testsuite/tests/lib-dynlink-initializers/test10_main.ml

```diff
@@ -0,0 +1,55 @@
+(* TEST
+
+include dynlink
+
+files = "test10_plugin.ml"
+
+libraries = ""
+
+* shared-libraries
+** setup-ocamlc.byte-build-env
+*** ocamlc.byte
+module = "test10_main.ml"
+*** ocamlc.byte
+module = "test10_plugin.ml"
```

Comment:
(I think you need `flags = "-g"` to have an interesting backtrace.)

Reply:
Thanks! That seems to fix the bytecode version.